lms/static/js/learner_dashboard/EnterpriseLearnerPortalModal.jsx

```diff
@@ -79,6 +87,16 @@ class EnterpriseLearnerPortalModal extends React.Component {
     return `${baseUrlWithSlug}?utm_source=lms_dashboard_modal`;
   }
 
+  handleLearnerPortalDashboardClick(e) {
+    e.preventDefault();
+    window.analytics.track('edx.ui.enterprise.lms.dashboard.learner_portal_modal.dashboard_cta.clicked', {
+      enterpriseUUID: this.props.enterpriseCustomerUUID,
+    });
+    setTimeout(() => {
+      window.location.href = this.getLearnerPortalUrl();
+    }, 300);
```

Comment:
Gives the analytics call ample time to resolve before we redirect off the page. This is similar to what `analytics.trackLink` does behind-the-scenes but opting not to try to use it here because it gets funky with creating duplicate event listeners when the component re-renders.